docs/onboarding-tech/2-product.md

```diff
@@ -0,0 +1,12 @@
+---
+date: 2024-02-12
+category: document
+author:  Samson Rinaldi
+order: 2
+---
+
+# Our Products
+
+Since 2015, Jangala has been developing and implementing internet access solutions to support the delivery of critical social services to marginalised communities. We utilise a variety of self-built and off-the-shelf techniques and technologies to ensure we can provide the best possible service to our users both now and in the future. Our products cater to a diverse range of needs, from easy-to-manage and scalable networks that can serve hundreds or even thousands of users, to at-home safe Wi-Fi that enables providers to monitor and manage connections.
+
+Along with our product offerings, we provide a range of services, including site assessments, network design, hardware support, partner training, and cloud infrastructure. Currently, our core products—Big Box, Get Box, and the soon-to-be-released Lock Box—form the foundation of our connectivity and content infrastructure. As we move forward, we remain dedicated to expanding our offerings by introducing new products and developing our existing solutions to meet the ever-changing needs of our users.
```

Comment:
For now, let's exclude Lock Box, until we're on firmer ground with it

Reply:
done